src/cgroup.c

```diff
@@ -306,33 +306,39 @@ gboolean check_cgroup2_oom()
 			continue;
 
 		if (counter != last_counter) {
-			if (write_oom_files() == 0)
+			if (create_oom_files() == 0)
 				last_counter = counter;
 		}
 		return G_SOURCE_CONTINUE;
 	}
 	return G_SOURCE_REMOVE;
 }
 
-/* write the appropriate files to tell the caller there was an oom event
- * this can be used for v1 and v2 OOMS
+static int create_oom_file(const char *base_path, const char *path_type)
+{
+	if (base_path == NULL || base_path[0] == '\0')
+		return 0;
+
+	_cleanup_free_ char *ctr_oom_file_path = g_build_filename(base_path, "oom", NULL);
+	_cleanup_close_ int ctr_oom_fd = open(ctr_oom_file_path, O_CREAT | O_CLOEXEC, 0666);
+	if (ctr_oom_fd < 0) {
+		nwarnf("Failed to write oom file to the %s path", path_type);
```

Comment:
If I read such a message using `ctr_oom_file_path` would make the most sense as a user is likely not knowing where this path would be and if there is a full file path a user could check this directory quickly.

Reply:
replaced `path_type` with `base_path` itself